core/controllers/reader.py

```diff
@@ -146,7 +176,17 @@ class ExplorationPageEmbed(base.BaseHandler):
 
     @require_playable
     def get(self, exploration_id):
-        """Handles GET requests."""
+        """Handles GET requests.
+        The require_playable decorator ensures the learner can play the
```

Comment:
Don't need to talk about this. Info about require_playable goes in the docstring for the decorator itself.